backend/tournesol/views/ratings.py

```diff
@@ -184,7 +185,20 @@ def get_queryset(self):
         return ratings
 
     def perform_create(self, serializer):
-        contributor_rating = serializer.save()
+        # In some cases, several concurrent requests might pass the
+        # serializer's validation and cause the view to fail here.
+        try:
+            contributor_rating = serializer.save()
+        except IntegrityError as err:
+            raise ValidationError(
```

Comment:
Maybe we could also add a warning here, just in case this `except` would catch other kinds of `IntegrityError` we don't expect.

Something like:
```python
        try:
            contributor_rating = serializer.save()
        except IntegrityError as err:
            logging.warning("Got IntegrityError when creating ContributorRating", exc_info=True)
            raise ValidationError(
            ...
```